codegen/templates/dependency_struct.tmpl

```diff
@@ -1,5 +1,5 @@
 {{$instance := . -}}
-package {{$instance.PackageInfo.PackageName}}
+package module
```

Comment:
since you fixed everything else to use `$instance.PackageInfo.PackageName` can this use `PackageInfo` again ?